Cleaning DB/1 Table cleaning.sql

```diff
@@ -0,0 +1,35 @@
+use CleaningDB 
+go 
+
+drop table if exists dbo.Customer 
+go
+create table dbo.Customer(
+    CustomerID int not null identity primary key,
+    FirstName varchar(20) not null 
+            constraint c_Customer_first_name_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(20) not null 
+            constraint c_Customer_last_name_cannot_be_blank check(LastName <> ''),
+    PhoneNum varchar(15) not null 
+            constraint c_Customer_phone_num_cannot_be_blank check(PhoneNum <> ''),
+    HomeAddress varchar(40) not null 
+            constraint c_Customer_home_address_cannot_be_blank check(HomeAddress <> '')
+            constraint u_Customer_home_address_must_be_unique unique,
+    Frequency varchar(10)
+            constraint c_Customer_frequency_must_be_weekly_or_biweekly check(Frequency in ('weekly', 'bi weekly')),
```

Comment:
I think biweekly is spelled as one word or bi-weekly

Reply:
Sorry, I just noticed that the data you were given spells it bi weekly, so you can just keep it like that